src/color.ts

```diff
@@ -113,14 +113,14 @@ const getProminent = (data: Data, args: Args): Output => {
   )
 }
 
-const process = (item: Item, args: Args, handler: Handler): Promise<Output> =>
+const process = (handler: Handler, item: Item, args?: Partial<Args>): Promise<Output> =>
```

Comment:
switched the position of the `handler` parameter since required parameters can't follow optional ones